lib/phlex/view/callbacks.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Phlex::View::Callbacks
+	def template(&block)
+		respond_to?(:before_template) && before_template
```

Comment:
I agree it shouldn't be `before_render`, since we have a `render` method but can we go with `before_rendering`? `template` is a noun and we can't do something before a noun. `before_rendering` would be short for `before_rendering_template`.

Reply:
Not sure doing something before a noun actually matters. I certainly don't think anyone will complain.

However, the important thing is that `before_template` is more accurate and says exactly what it does, whereas `before_rendering` is kinda misleading, and doesn't actually imply it is called before the template.

Reply:
I’m happy to go with `before_rendering_template` to be really clear.

Reply:
Compromise 👌

Reply:
What about `after_template`? Not sure "rendering" should be used for that one, as it's not actually called after rendering.

Reply:
It’s executed in the `call` method which is essentially a "render_template" method.